core/cmd/client.go

```diff
@@ -695,9 +695,9 @@ func (t *promptingAPIInitializer) Initialize(orm sessions.ORM) (sessions.User, e
 			email := t.prompter.Prompt("Enter API Email: ")
 			pwd := t.prompter.PasswordPrompt("Enter API Password: ")
 			// On a fresh DB, create an admin user
-			user, err := sessions.NewUser(email, pwd, sessions.UserRoleAdmin)
-			if err != nil {
-				t.lggr.Errorf("Error creating API user: ", err, "err")
+			user, err2 := sessions.NewUser(email, pwd, sessions.UserRoleAdmin)
+			if err2 != nil {
+				t.lggr.Errorf("Error creating API user: ", err2, "err")
```

Comment:
This looks like it was already broken. `Errorf`? Shouldn't it be `Error`, or:
```suggestion
				t.lggr.Errorw("Error creating API user", "err", err2)
```